interop/tapir/src/main/scala/caliban/interop/tapir/TapirAdapter.scala

```diff
@@ -149,12 +132,10 @@ object TapirAdapter {
             excludeExtensions = cacheDirective.map(_ => Set(Caching.DirectiveName))
           )
         )
-      case resp if accepts == GraphqlServerSentEvent.mediaType  =>
-        val code = response.errors.collectFirst { case HttpRequestMethod.MutationOverGetError => StatusCode.BadRequest }
-          .getOrElse(StatusCode.Ok)
```

Comment:
@SvenW I missed this one in the previous PR. [Reading the spec](https://github.com/enisdenjo/graphql-sse/blob/master/PROTOCOL.md), I think we shouldn't do this for SSE (although I don't see how we would get a mutation over SSE anyways):

> Additionally, responding with a 400 (Bad Request) will cause the user agent to fail the connection. In some cases, like with the browser's native EventSource, the error event will hold no meaningful information helping to understand the validation issue(s).

Reply:
Yeah. Sounds like 200 is the only valid code to send